functions.py

```diff
@@ -0,0 +1,84 @@
+from math import remainder
+from datetime import datetime
+from typing import List
+from .readJSON import Myjson
+from config import day, MessageType
+from mail import Mail
+
+
+def isToday(d : day, time = None):
+    """
+    Input: @time - integer constant indicating an hour 0 - 23
+           @d - day enum 
+    Output: True if current time matches day and hour, else False.
+    """
+    if time == None:
+        return datetime.today().weekday() == day.value
+    else:
+        return datetime.today().weekday() == day.value and \
+            time == datetime.today().hour
+
+def alert(data , msg_type : MessageType):
+    """
+    Input: @data - relevant data for send a message.
+           @msg_type - message type enum
+    Calls 'createalert' according to data
+    """
+    # Case where data is combined out of 2 different names (sadir soliders)
+    if type(data) == tuple:
+        createAlert(data[0], msg_type)
```

Comment:
How does this even work? You are calling create alert with data, that is a string, but it's expecting a mail. Then you even try to call a function on this string, so either I don't get something or you haven't checked your code.

Reply:
I now notice that `createAlert` expects 3 arguments and you're only passing 2.

Reply:
My bad, fixed it. An argument was indeed missing. 
Data is either a tuple/ string/list, depends on the message type.